framework/src/modules/interoperability/base_cc_commands/registration.ts

```diff
@@ -38,16 +38,30 @@ export abstract class BaseCCRegistrationCommand<
 	}
 
 	public async verify(ctx: ImmutableCrossChainMessageContext): Promise<void> {
-		const { ccm } = ctx;
+		const { ccm, ccu } = ctx;
 		if (!ccm) {
 			throw new Error('CCM to execute registration cross chain command is missing.');
 		}
+		if (!ccu) {
+			throw new Error('CCU to execute registration cross chain command is missing.');
+		}
```

Comment:
I think if they are mandatory fields coming from ctx then we can safely assume that they should be present so we can remove these conditions